packages/core/src/components/slider/rangeSlider.tsx

```diff
@@ -44,6 +44,9 @@ export interface IRangeSliderProps extends ISliderBaseProps {
 
     /** Callback invoked when a handle is released. */
     onRelease?(value: NumberRange): void;
+
+    /** Applied to `aria-label` props of the slider Handles */
+    handleAriaLabels?: { startAriaLabel: string; endAriaLabel: string };
```

Comment:
following my comment on `slider.tsx`... my API suggestion here would be

```suggestion
    /** Subset of handle props to apply to the start and end Handles. */
    handleProps?: {
    	start: Pick<HandleProps, "ariaLabel" | "ariaLabelledBy">;
    	end: Pick<HandleProps, "ariaLabel" | "ariaLabelledBy">;
    };
```

Reply:
yep, see new changes